app/views/shared/_citations.html.erb

```diff
@@ -0,0 +1,57 @@
+<% # OVERRIDE Hyraxv5.0.5 to guard against nil keyword %>
```

Comment:
Is it possible that any of the others might be nil or even undefined with flexible metadata?

Reply:
fix incoming